sample/src/main/java/com/github/paylike/sample/sample.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.paylike.sample
+
+import com.github.paylike.kotlin_client.PaylikeClient
+import kotlinx.coroutines.runBlocking
+
+fun main()
+{
+    val client = PaylikeClient("TestID01")
+    runBlocking {
+        println("hello")
```

Comment:
Does not seem like a complete usage example 😄

Reply:
Yeah, the commit message says: added implementation and tests; so sample is still missing

Reply:
done